api/can_api_v2_definition.py

```diff
@@ -368,6 +374,9 @@ class Annotations(base_model.APIBaseModel):
             "Ratio of population that are fully vaccinated and have received a booster (or additional) dose."
         ),
     )
+    vaccinationsBivalentDoseRatio: Optional[FieldAnnotations] = pydantic.Field(
+        None, description="Ratio of population have received a bivalent vaccine dose."
```

Comment:
should be "population **that** have" I think?

But actually, should this and vaccinationsAdditionalDoseRatio be changed to "Annotations for vaccination...DoseRatio"? I think maybe we copy/pasted an incorrect description.

Reply:
oops, yeah thanks!